pinot-core/src/main/java/org/apache/pinot/core/plan/FilterPlanNode.java

```diff
@@ -122,8 +127,29 @@ private BaseFilterOperator constructPhysicalOperator(FilterContext filter,
           DataSource dataSource = _indexSegment.getDataSource(lhs.getIdentifier());
           switch (predicate.getType()) {
             case TEXT_MATCH:
-              return new TextMatchFilterOperator(dataSource.getTextIndex(),
-                  ((TextMatchPredicate) predicate).getValue(), _numDocs);
+                return new TextMatchFilterOperator(
```

Comment:
Why is the formatting getting changed here?

Reply:
restored.